common/src/main/java/com/wynntils/features/user/TooltipScaleFeature.java

```diff
@@ -19,9 +20,9 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @EventListener
-@Configurable(category = "Item Tooltips")
+@ConfigurableInfo(category = "Item Tooltips")
 @FeatureInfo(stability = Stability.STABLE)
-public class TooltipScaleFeature extends UserFeature {
+public class TooltipScaleFeature extends UserFeature implements Configurable {
```

Comment:
Maybe `UserFeature` itself could implement `Configurable`

Reply:
not a bad idea!